src/migrations/20200101101824-create-request.js

```diff
@@ -14,6 +14,18 @@ export const up = (queryInterface, Sequelize) => queryInterface.createTable('Req
   status: {
     type: Sequelize.STRING
   },
+  tripType: {
```

Comment:
@higustave-ops always create a new migration file instead of altering the existing one.